api/users.go

```diff
@@ -155,3 +155,11 @@ func (c *Client) SendConsentCodeSMS(accessToken string, realmName string, userID
 	_, err := c.post(accessToken, nil, url.Path(sendSmsConsentCode), url.Param("realm", realmName), url.Param("userId", userID))
 	return err
 }
+
+// GetTermsOfUseDeletableUsers gets the list of users created for a
+// long time (configured in Keycloak) who declined the terms of use
+func (c *Client) GetTermsOfUseDeletableUsers(accessToken string) ([]keycloak.DeletableUserRepresentation, error) {
+	var deletableUsers []keycloak.DeletableUserRepresentation
+	err := c.get(accessToken, &deletableUsers, url.Path(adminExtensionAPIPath), url.Param("realmReq", "master"), url.Param("realm", "terms-of-use-denied-users"))
```

Comment:
Check if we can publish this API without /reamls/... if no possible use a ficitive "all" realms to make it more comprehensible